README.md

```diff
@@ -510,7 +510,12 @@ You can specify config defaults that will be applied to every request.
 
 ```js
 axios.defaults.baseURL = 'https://api.example.com';
+axios.defaults.headers.common['User-Agent'] = /* eg. 'AppName/1.0.0' */;
+
+// nb: If axios is used with multiple domains, the AUTH_TOKEN will be sent to all of them.
```

Comment:
Can we change `nb: ` to read as `Important: `

Reply:
Yep, no problem